FBTweak/_FBTweakCategoryViewController.m

```diff
@@ -70,10 +70,9 @@ - (void)viewDidLoad
   self.navigationItem.rightBarButtonItem = [[UIBarButtonItem alloc] initWithBarButtonSystemItem:UIBarButtonSystemItemDone target:self action:@selector(_done)];
   
   if ([MFMailComposeViewController canSendMail]) {
```

Comment:
this whole thing is not needed. remove the `MessageUI.h` import and drop anything related.

Reply:
Done.